### ISSUE TITLE
"No such file or directory" for latest release

### ISSUE THREAD

**Issue by codegain** [djfdyuruiry/improved-yarn-audit#19]
Hi,

I'm getting the following after upgrading to the latest version on Linux:

```
No such file or directory
error Command failed with exit code 127.
```

I also added a blank `.iyarc` file to see if this fixes it, but it's still showing the error. After downgrading to 2.2.1, the error goes away.

**Comment by drtyh2o** [comment]
I am getting the following error on MacOS when running `yarn improved-yarn-audit`
```sh
env: node\r: No such file or directory
```
I believe this is because the latest version (supporting WIndows) was published with CRLF line endings which don't play nicely with `yarn`.

As a workaround, `npx improved-yarn-audit` works.

**Comment by djfdyuruiry** [comment]
Hey Guys,

My bad - was not aware that npm doesn't do this, will publish from *nix going forward!

Fixed in v2.3.1
